hapi/hedera-protobuf-java-api/src/main/proto/platform/state/virtual_map_state.proto

```diff
@@ -270,6 +285,317 @@ message VirtualMapKey {
+     * A state identifier for scheduled throttle usage snapshots.
+     */
+    proto.ThrottleUsageSnapshots ScheduleService_I_SCHEDULED_USAGES = 34;
+
+    /**
+     * A state identifier for the TSS encryption keys.
+     */
+    com.hedera.hapi.node.state.tss.TssEncryptionKeys TssBaseService_I_TSS_ENCRYPTION_KEY = 35;
+
+    /**
+     * A state identifier for hinTS key sets.
+     */
+    com.hedera.hapi.node.state.hints.HintsKeySet HintsService_I_HINTS_KEY_SETS = 37;
+
+    /**
+     * A state identifier for the active hinTS construction. Singleton state.
+     */
+    com.hedera.hapi.node.state.hints.HintsConstruction HintsService_I_ACTIVE_HINTS_CONSTRUCTION = 38;
+
+    /**
+     * A state identifier for the next hinTS construction. Singleton state.
+     */
+    com.hedera.hapi.node.state.hints.HintsConstruction HintsService_I_NEXT_HINTS_CONSTRUCTION = 39;
+
+    /**
+     * A state identifier for hinTS preprocessing output votes.
+     */
+    com.hedera.hapi.node.state.hints.PreprocessingVote HintsService_I_PREPROCESSING_VOTES = 40;
+
+    /**
+     * A state identifier for the entity counts. Singleton state.
+     */
+    com.hedera.hapi.node.state.entity.EntityCounts EntityIdService_I_ENTITY_COUNTS = 41;
+
+    /**
+     * A state identifier for the ledger id. Singleton state.
+     */
+    proto.ProtoBytes HistoryService_I_LEDGER_ID = 42;
+
+    /**
+     * A state identifier for history proof key sets.
+     */
+    com.hedera.hapi.node.state.history.ProofKeySet HistoryService_I_PROOF_KEY_SETS = 43;
+
+    /**
+     * A state identifier for the active proof construction. Singleton state.
+     */
+    com.hedera.hapi.node.state.history.HistoryProofConstruction HistoryService_I_ACTIVE_PROOF_CONSTRUCTION = 44;
+
+    /**
+     * A state identifier for the next proof construction. Singleton state.
+     */
+    com.hedera.hapi.node.state.history.HistoryProofConstruction HistoryService_I_NEXT_PROOF_CONSTRUCTION = 45;
+
+    /**
+     * A state identifier for signatures on roster transition histories.
+     */
+    com.hedera.hapi.node.state.history.RecordedHistorySignature HistoryService_I_HISTORY_SIGNATURES = 46;
+
+    /**
+     * A state identifier for votes on history proofs.
+     */
+    com.hedera.hapi.node.state.history.HistoryProofVote HistoryService_I_PROOF_VOTES = 47;
+
+    /**
+      * A state identifier for the CRS state. Singleton state.
+      */
+    com.hedera.hapi.node.state.hints.CRSState HintsService_I_CRS_STATE = 48;
+
+    /**
+     * A state identifier for the CRS publications.
+     */
+    com.hedera.hapi.services.auxiliary.hints.CrsPublicationTransactionBody HintsService_I_CRS_PUBLICATIONS = 49;
+
+    /**
+     * A state identifier for the node rewards. Singleton state.
+     */
+    proto.NodeRewards TokenService_I_NODE_REWARDS = 50;
+
+    /**
+     * Metadata of the round receipts queue.
+     */
+    // TODO: double check
+    proto.TransactionReceiptEntries RecordCacheTransactionReceiptQueue = 126;
```

Comment:
This looks correct, but please double check with @jasperpotts and @jsync-swirlds